pipeline/inputs/node-exporter-metrics.md

```diff
@@ -58,20 +58,20 @@ The following table describes the available collectors as part of this plugin. A
 
 > note: the Version column specifies the Fluent Bit version where the collector is available.
 
-| Name      | Description                                                                                      | OS     | Version |
-| --------- | ------------------------------------------------------------------------------------------------ | ------ | ------- |
-| cpu       | Exposes CPU statistics.                                                                          | Linux  | v1.8    |
-| cpufreq   | Exposes CPU frequency statistics.                                                                | Linux  | v1.8    |
-| diskstats | Exposes disk I/O statistics.                                                                     | Linux  | v1.8    |
-| filefd    | Exposes file descriptor statistics from `/proc/sys/fs/file-nr`.                                  | Linux  | v1.8.2  |
-| loadavg   | Exposes load average.                                                                            | Linux  | v1.8    |
-| meminfo   | Exposes memory statistics.                                                                       | Linux  | v1.8    |
-| netdev    | Exposes network interface statistics such as bytes transferred.                                  | Linux  | v1.8.2  |
-| stat      | Exposes various statistics from `/proc/stat`. This includes boot time, forks, and interruptions. | Linux  | v1.8    |
-| time      | Exposes the current system time.                                                                 | Linux  | v1.8    |
-| uname     | Exposes system information as provided by the uname system call.                                 | Linux  | v1.8    |
-| vmstat    | Exposes statistics from `/proc/vmstat`.                                                          | Linux  | v1.8.2  |
-| systemd collector | Exposes statistics from systemd.                                                           | Linux  | v2.1.3  |
+| Name      | Description                                                                                      | OS           | Version      |
+| --------- | ------------------------------------------------------------------------------------------------ | ------------ | ------------ |
+| cpu       | Exposes CPU statistics.                                                                          | Linux,macOS  | v1.8,v2.2    |
```

Comment:
Should this be 2.2 version @cosmo0920 ?

Reply:
Yes. macOS metrics should be in v2.2 if we ship in the near future. This feature is not targeted for v2.1.